README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# HW1
+
+# About me
+My name is **Chenyang He**. I am a MAPSS student concentrating in *Psychology*. 
+
+# Research Interest 
```

Comment:
Hi Chenyang! I really liked the formatting of your biography; I like the way you use 3 headers to clearly structure the information you provide, and how, within your "Research Interest" section you use an ordered list to further organize the questions that your lab researches.